lib/helpers/create-winston-logger.helper.ts

```diff
@@ -110,3 +128,11 @@ function getMeta(meta: any): string {
 
     return ``
 }
+
+function getMetaForProduction(meta: any): any {
+    if (typeof meta[0] === 'string' && meta[0].length === 1) {
```

Comment:
I still don't get the idea of this `if` could you clarify what it does especially `meta[0].length === 1`?

Reply:
NestJS logs look like this: logger.log(`ModuleName initialized`, `InstanceLoader`), so the second arg splits to array and this is shit. This `if` dont let you logs string in meta

Reply:
format.splat() from winston splits string to array